app/styles/partials/welcome.scss

```diff
@@ -26,3 +26,7 @@
     }
   }
 }
+
+.main-container .main-heading{
```

Comment:
@gr8shivam Why do we need `main-container` here? Why are we adding the color to `main-container`?

Reply:
We aren't adding color to the main-container. It's css-child targeting. It's just to make sure that main-heading inside the main-container is targeted, so it doesn't conflict with any other element outside main-container that may have a main-heading class.

Reply:
Waiting for your reply on this. @geekyd

Reply:
@gr8shivam My bad but we are not using the main-heading class anywhere else in the project so it is safe to apply css to a single class(main-heading) . Also please add a space before the starting brace it is leading to failure of es-lint test for this file. Check https://travis-ci.org/fossasia/open-event-frontend/builds/287866115#L1453